notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>I'm curious how well would have an <a href="https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.interpolate.html" target="_blank">interpolation</a> performed in this scenario? Do you feel like this is also a method that should be added for comparison?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Not sure about interpolation however I am thinking of adding MICE imputation in the next iteration.</p>